site/src/components/BlockCard.tsx

```diff
@@ -79,7 +79,7 @@ export const BlockCard: VFC<BlockCardProps> = ({ loading, data }) => {
   } = data;
 
   return (
-    <Link href={`${packagePath}`}>
+    <Link href={`/${packagePath}`}>
```

Comment:
Follows #44

Blocks used relative path (i.e. `@hash/abc`) and were classified as external because of that. Adding `/` ensures that they are not opened in the new tab (which is annoying).